nuc_morph_analysis/lib/preprocessing/global_dataset_filtering.py

```diff
@@ -199,6 +204,9 @@ def process_all_tracks(df, dataset, remove_growth_outliers, num_workers):
     df = add_fov_touch_timepoint_for_colonies(df)
     df = add_features.add_non_interphase_size_shape_flag(df)
     df = add_change_over_time(df)
```

Comment:
Loading the data is failing at this line so I wont be able to test any of the code. 

 ```
   205     df = add_features.add_non_interphase_size_shape_flag(df)
--> 206     df = add_change_over_time(df)
    207     df = add_volume_change_over_25_minute_window(df)
    208 
    209     # df = add_dvdt_over_V(df)

...

ValueError: periods must be an integer
```

Reply:
Thank you!! Reviewing and running things now!